deploy/crds/pulumi.com_programs.yaml

```diff
@@ -0,0 +1,132 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.9.2
+  creationTimestamp: null
+  name: programs.pulumi.com
+spec:
+  group: pulumi.com
+  names:
+    kind: Program
+    listKind: ProgramList
+    plural: programs
+    singular: program
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
+    name: v1
```

Comment:
Would there be any reason to add this CRD as a pre-v1 version, or are you sure about the API?

Reply:
We talked about this a bit when pairing on it - I think this is fine, but I'm happy to move it if @squaremo thinks it would be wise to do so.

Reply:
There may be precedents indicating otherwise, but I think generally APIs don't get changed once there's a new version. It'd be a bit odd for an API to depend on -- well, refer to -- a less stable API (certainly one of the same group).

The conservative way to do it would be to introduce a new, unstable version of the API (e.g., v2alpha1) and add the Program type, and change the Stack type, in that. As discussed [elsewhere](https://github.com/pulumi/pulumi-kubernetes-operator/pull/324#discussion_r975792786), expedience puts a thumb on the scales.